docker/files/usr/local/etc/gunicorn/richie.py

```diff
@@ -1,8 +1,14 @@
 # Gunicorn-django settings
 bind = ['0.0.0.0:8000']
-graceful_timeout = 90
-loglevel = 'error'
 name = 'richie'
 python_path = '/app'
+
+# Run
+graceful_timeout = 90
 timeout = 90
 workers = 3
+
+# Logging
+capture_output = True
+errorlog = '-'
```

Comment:
Good point! I've added the `accesslog` configuration so that we redirect access logs to `stdout` and error logs to `stderr`, as recommended in Docker's documentation [1].

[1] https://docs.docker.com/config/containers/logging/